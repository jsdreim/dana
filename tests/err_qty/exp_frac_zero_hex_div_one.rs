#![allow(unused_imports)]
use dana::{qty, symbols::*};

fn main() {
    let _err = qty![1.0 m^(0x00/1)];
}
