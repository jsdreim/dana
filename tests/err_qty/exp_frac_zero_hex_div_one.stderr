error: unit with exponent of zero is scalar
 --> tests/err_qty/exp_frac_zero_hex_div_one.rs:5:28
  |
5 |     let _err = qty![1.0 m^(0x00/1)];
  |                            ^^^^
